addons/calendar_sms/models/calendar.py

```diff
@@ -27,6 +27,19 @@ def _do_sms_reminder(self):
                 put_in_queue=False
             )
 
+    def action_send_sms(self):
```

Comment:
Why doesn't this one raise an error if not recipients ? Otherwise this one seems good as it targets partners.